code/drasil-gool/lib/GOOL/Drasil/AST.hs

```diff
@@ -107,13 +107,14 @@ updateParam :: (Doc -> Doc) -> ParamData -> ParamData
 updateParam f v = pd (paramVar v) ((f . paramDoc) v)
 
 -- Used as the underlying data type for Programs in all renderers
-data ProgData = ProgD {progName :: String, progPurp :: String, progMods :: [FileData]}
+data ProgData = ProgD {progName :: String, progPurp :: String, 
+  drasilVers :: String, progMods :: [FileData]}
 
-progD :: String -> String -> [FileData] -> ProgData
-progD n st fs = ProgD n st (filter (not . isEmpty . modDoc . fileMod) fs)
+progD :: String -> String -> String -> [FileData] -> ProgData
```

Comment:
Here is where the unbundled arguments have gone crazy. Above the threshold that I can bear.